src/site/content/en/handbook/quick-start/index.md

```diff
@@ -11,7 +11,7 @@ description: |
 Content creation for web.dev has three phases: planning, writing, and publishing.
 
 ## Planning
-1. Start a content proposal issue on GitHub using [this issue template](https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=content+request&template=content_request.md&title=). The checklist in your issue will guide you through the rest of the process.
+1. Start a content proposal issue on GitHub using [this issue template](https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=content+request&template=content_request.md&title=). The checklist in your issue guides you through the rest of the process.
 1. Create a copy of [this document template](https://docs.google.com/document/d/1ydauCufwwavStaKxhIDHuNivVxgQBOdDno4uLjGIjmE/edit?usp=sharing) to briefly write up your content idea. Put the link to your doc in your GitHub issue.
 1. The web.dev team will take a look to see if the idea fits with the [goals of the site](/about). If the idea is approved, it gets slated for a publication date!
```

Comment:
This process doc uses future tense, which is against the dev style guide: https://developers.google.com/style/tense

Would you be against saying "The web.dev team takes a look to see if the idea..." and "The tech writer reviews your doc using the checklist..."?

Reply:
He definitely should. Here's why.

* Present tense follows the writing best practice of omitting needless words. ('The system will respond' vs. 'The system responds'.)
* The handbook needs to model best practices.

Reply:
So, I've changed these and some other instances of future tense, but I've kept others because changing them to present would break the logic of the sentence since some events are sequenced later relative to others.

One other thing, FWIW: I'm not totally convinced by the rationales for always using present, especially for events that happen as a result of commands. For example, the Letter from the Editors linked on the cited GDDSC page argues that auxiliary verbs add cognitive load. But given that (a) readers' eyes tend to fixate on contentful words and (b) we're reducing explicit sequencing information, I wonder if that's true. Would be interesting to poke around on ERIC to see if somebody's done a study.